odoo_backup_sh/models/odoo_backup_sh.py

```diff
@@ -41,7 +41,7 @@ class BackupConfig(models.Model):
     _description = 'Backup Configurations'
     _rec_name = 'database'
 
-    DATABASE_NAMES = [(db, db) for db in odoo.service.db.list_dbs() if db != 'session_store']
+    DATABASE_NAMES = [(db, db) for db in odoo.service.db.list_dbs(force=True) if db != 'session_store']
```

Comment:
Actually, I'm not sure about this solution. Imagine we have different companies that share the same odoo instance. Those companies are allowed to use backup module. Then any admin can get backup of all companies.

What we can do istead is use current database only when list_db is disabled

Reply:
> What we can do i[n]stead is use current database only when list_db is disabled

Doing like this then: https://github.com/odoo/odoo/blob/8f4b98e67fe49516c0c830524de351db67c58357/odoo/service/db.py#L369